clients/ptranslator/src/main/java/org/hawkular/metrics/clients/ptrans/ConfigurationKey.java

```diff
@@ -1,87 +0,0 @@
-/*
- * Copyright 2014-2016 Red Hat, Inc. and/or its affiliates
- * and other contributors as indicated by the @author tags.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.hawkular.metrics.clients.ptrans;
-
-/**
- * PTrans configuration keys.
- *
- * @author Thomas Segismont
- */
-public enum ConfigurationKey {
```

Comment:
Good catch.

I dropped it as I was completely rewriting the property names (this 
helped to manually test faster). And then I forgot to add it back (and 
updated).

Will do

Le 07/07/2016 à 19:18, Stefan Negrea a écrit :
> In
> clients/ptranslator/src/main/java/org/hawkular/metrics/clients/ptrans/ConfigurationKey.java
> <https://github.com/hawkular/hawkular-metrics/pull/529#discussion_r69947951>:
>
>> - *    http://www.apache.org/licenses/LICENSE-2.0
>> - *
>> - * Unless required by applicable law or agreed to in writing, software
>> - * distributed under the License is distributed on an "AS IS" BASIS,
>> - * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
>> - * See the License for the specific language governing permissions and
>> - * limitations under the License.
>> - */
>> -package org.hawkular.metrics.clients.ptrans;
>> -
>> -/**
>> - * PTrans configuration keys.
>> - *
>> - * @author Thomas Segismont
>> - */
>> -public enum ConfigurationKey {
>
> Why did you drop this class?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hawkular/hawkular-metrics/pull/529/files/f1db90b1cb3cdaebf6be5bfffd50a71e8182e2bc..0238f7d7e79b5236c6acebd6e6e7c199c51a6d42#r69947951>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABbltqSx4R3ESoTcqiUxf9C1AxyYDyWoks5qTTTSgaJpZM4JGMbv>.
>

-- 
Thomas Segismont
JBoss ON Engineering Team